testsuites/syncgateway/functional/tests/test_db_online_offline.py

```diff
@@ -188,9 +188,18 @@ def test_online_to_offline_changes_feed_controlled_close_continuous(params_from_
         futures = dict()
         futures[executor.submit(seth.start_continuous_changes_tracking, termination_doc_id=None)] = "continuous"
         futures[executor.submit(doc_pusher.add_docs, num_docs)] = "docs_push"
-        time.sleep(5)
-        futures[executor.submit(sg_client.take_db_offline, cluster_conf, "db")] = "db_offline_task"
-
+        offline_retries = 0
+        time_sec = 5
+        while offline_retries < 5:
+            try:
+                assert sg_client.take_db_offline(cluster_conf, "db") == 0
+                futures[executor.submit(sg_client.take_db_offline, cluster_conf, "db")] = "db_offline_task"
+                break
+            except AssertionError as error:
+                offline_retries = offline_retries + 1
+                time.sleep(time_sec * offline_retries)
```

Comment:
every retry is getting multiplied with time_sec. do we require that. we can have static time.sleep with 5 seconds should be good.